example/lib/model/ble_peripheral.dart

```diff
@@ -6,8 +6,27 @@ class BlePeripheral extends Equatable {
   final int rssi;
   final bool isConnected;
 
+  BlePeripheralCategory get category =>
+      BlePeripheralCategoryResolver.categoryForName(name);
```

Comment:
Actually, why is this a dynamic getter? Perhaps BlePeripheral should just get in the contructor and BLoC, or whatever it is that creates it, should decide what category is peripheral x.

Reply:
Changed `category` to a final field.